### DISCUSSION TITLE
Image fails to upload without any error message

### DISCUSSION THREAD

**Discussion by maxime1992** [photoprism/photoprism#3915] [Q&A]
#### 1. What is not working as documented?

Says upload is successful, but the picture is never there?

#### 2. How can we reproduce it?

Try to upload this picture:

![DSCN1263](https://github.com/photoprism/photoprism/assets/4950209/0a95e72f-c602-42eb-8a23-e452a790649a)

(MD5 should be `ebaf7b60146fc62c0d1f4da1b84083c0`)

It'll upload successfully but the picture will never be visible anywhere.

#### 3. What behavior do you expect?

Have the picture correctly uploaded.

#### 4. What could be the cause of your problem?

I know that I've had issues in the past with Samsung panoramas but it was fixed by I believe applying a transformation using image magick (?) if the processing failed or something like that. Maybe this is kind of similar?

#### 5. Can you provide us with example files for testing, error logs, or screenshots?

I have provided a file above.

#### 6. Which software versions do you use?

Build 231021-9abea5b55

OS: Ubuntu 22.04.3 LTS (GNU/Linux 5.15.0-88-generic x86_64)

DB: `mariadb:10.11`

#### 7. On what kind of device is PhotoPrism installed?

Old laptop on an SSD, but probably unrelevant here.

#### 8. Do you use a Reverse Proxy, Firewall, VPN, or CDN?

Unrelevant as other pics upload just fine.

**Comment by lastzero** [comment]
Please check your logs, increase the log level if needed, and go through the first following checklist step by step:
- https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

**Comment by lastzero** [comment]
Works for me:

![works-for-me](https://github.com/photoprism/photoprism/assets/301686/df4cd30f-3eb3-4e91-888c-51394127462d)

@maxime1992 As detailed in the [checklist linked above](https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures), there can be many reasons why the image is not displayed as expected, e.g. it could be in *Private* or it's a duplicate. Since bug reports are not meant to find out if a problem is an actual bug in our software or has another cause, I will convert this into a discussion until it is confirmed/reproducible and it is clear that this is something that needs to be resolved in our code.

**Reply by maxime1992** [reply]
@lastzero I dug a bit more. Turns out for the fail import that now it seems to be a duplicate. BUT, if I keep uploading this picture, it's because I cannot find it anywhere.

I have checked directly from my server, I can see the original picture here:

![image](https://github.com/photoprism/photoprism/assets/4950209/d5a2cda2-d61d-4780-b31e-cf442038ec99)

It's in the 2012/05 folder of photoprism original folder. BUT when I go from the app in library/originals/2012/05, I can see all the pics around that one, but not that one for some reason.

If that helps in any way, this is the case for all the vertical pics from that album for some reason :shrug:.

Any idea why those vertical pics would end up as expected in the originals folder on disk but not being shown in the UI?

**Reply by graciousgrey** [reply]
Currently, duplicates are not displayed in PhotoPrisms UI. Only the image that was indexed first is displayed. A user interface for displaying duplicates is planned:

- https://github.com/photoprism/photoprism/issues/1308

I thought we also had a ticket for displaying duplicates only in Originlas, but I can't find it offhand.

**Reply by maxime1992** [reply]
I'm not trying to display a duplicate, I'm trying to display the original picture. The original picture is not displayed anywhere. Whether I go directly in the original folder using photoprism UI, using the "Calendar" view from the menu, using the search one... 

> Only the image that was indexed first is displayed

So no, in my case, the original isn't displayed, that's my issue. I'm not trying to display a duplicate.

**Reply by graciousgrey** [reply]
Sorry, I misunderstood your previous comment. Can you try to reindex this folder and watch the logs? Ideally, you set the log level to trace beforehand. 

Could it be that the files are stacked with files from another folder?

If you haven't already done so, see also:
https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

**Reply by graciousgrey** [reply]
You can also try to search the files using `filename:"*yourfilename.jpg" quality:0` this should display them even if they are stacked or in review.

**Reply by maxime1992** [reply]
Umh, without even starting the re-indexing I can see this

![image](https://github.com/photoprism/photoprism/assets/4950209/7f99e81e-1211-4f25-9626-75e46b349458)

and I'm launching a re-index with the following settings

![image](https://github.com/photoprism/photoprism/assets/4950209/31cc7a11-2180-44c6-b3b0-7e04b9b7c9cf)

**Reply by maxime1992** [reply]
Looking in the logs for the picture that's missing, the only line that popped up is this one:

```
time="2023-12-07T12:44:26+01:00" level=info msg="index: updated main jpg file 2012/05/20120512_150757_76EAD7EF.jpg"
```

so nothing suspicious.. and it looks like it was happy overall

```
time="2023-12-07T12:47:10+01:00" level=error msg="index: already running"
time="2023-12-07T12:47:10+01:00" level=info msg="index: updated 183 files [3m5.197378018s]"
time="2023-12-07T12:47:18+01:00" level=info msg="indexing completed in 192 s"
```

**Reply by maxime1992** [reply]
Interesting though! It does show up with the filter you mentioned

![image](https://github.com/photoprism/photoprism/assets/4950209/45354c80-c169-4067-b0d2-44775da30f42)

**Reply by maxime1992** [reply]
Uuuuuuuuuuuuh, looks like the date is wrong for some reason. Could be this! :exploding_head: 

Why would the date be wrong only for those vertical pics, I've got no ideas though but it's probably that, let me investigate a little bit further (later) but don't bother meanwhile. I guess it comes from that. Thanks a lot for the trick to find the pic by filename
